CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+#Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other standard  method with the owners of this repository before making a change.
+
+## Pull Request process
```

Comment:
https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/CONTRIBUTING.md
add code snippets to the file.
Grading sheet
you  can add gif's too